04-patterns.Rmd

```diff
@@ -53,13 +53,14 @@ Don’t over explain if the copy already mentions a fact. The subhead, callout,
 ### Color
 Use a single color to represent the same type of data. Watch out for positive and negative numbers. Don’t use red for positive numbers or green for negative numbers. Those color associations are so strong it will automatically flip the meaning in the viewer’s mind. Make sure there is sufficient contrast between colors. Avoid patterns. Stripes and polka dots sound fun, but they can be incredibly distracting. If you are trying to differentiate, say, on a map, use different saturation of the same color. On that note, only use solid-colored lines (not dashes). Select colors appropriately. Don’t use more than 6 colors in a single layout.
 
-**- Leveraging Color in Visuals**
+**- Tips for Color in Visuals**
 
 |Use Case | Tip                   | Rationale                      |
 | ------- | ----------------------| ------------------------------ |
 Numerical Scales | Color for numerical scales should be used with caution. | The way you interpret a shade depends on the colors around it and sometimes it can lead to false conclusions. |
 Color Associations | Color can be used to leverage long-term memory very quickly. | We automatically associated strawberries with red. If we can leverage the how people associate different colors with different things, we will not even need a legend to explicitly match color to meaning. |
 Highlights | Bright colors can be used to highlight a certain part of the data. |Alarming colors draw the eye quickly to areas that need attention. |
+Color Combinations | Combinations like red–green or blue– yellow should be avoided. | This will cause difficulty for people with color blindness | 
```

Comment:
Hey, I think it's a really important addition.

However, you can add more color combinations like red-green, green-brown, green-blue, blue-grey, blue-purple, green-grey and green-black to this tip.

Ref - https://venngage.com/blog/color-blind-friendly-palette/

Reply:
A good resource! Thank you so much! 
But I think it's better to just add two examples because, in the table, the contents should be balanced for a better reading experience(not too long or too short), and these contents are from other citation.  However,  it is nice to add this resource as a reference! I'll change that!